catroid/src/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -115,6 +130,55 @@ public void onCreate(Bundle savedInstanceState) {
 		CameraManager.getInstance().setStageActivity(this);
 	}
 
+	private void setupAskHandler() {
+		messageHandler = new Handler(Looper.getMainLooper()) {
+			@Override
+			public void handleMessage(Message message) {
+				ArrayList<Object> params = (ArrayList<Object>) message.obj;
+
+				if (message.what == ASK_MESSAGE) {
+					showDialog((String) params.get(1), (AskAction) params.get(0));
+				}
+			}
+		};
+	}
+
+	private void showDialog(String question, final AskAction askAction) {
+		pause();
```

Comment:
you may have confused StageActivity and StageListener. This is the StageActivity class, so StageActivity.pause() is called, where vibration is stopped as well ;)
I tested it as well: Both Vibration and flash are paused during the dialogue is shown and resumed afterwards. Thank you though for reviewing my code in such a detail! :)